k8s-handle.py

```diff
@@ -109,12 +113,33 @@ def main():
 
         if 'use_kubeconfig' in args and args.use_kubeconfig:
             load_kube_config()
```

Comment:
Can we read namespace from library object?

Reply:
Thanks, good point.